manifests/dev.pp

```diff
@@ -27,7 +27,12 @@
   }
 
   if $::operatingsystem == 'Ubuntu' {
-    ensure_packages(["${php::globals::package_prefix}xml"])
+    if ( ! defined(Package["${php::globals::package_prefix}xml"])) {
```

Comment:
is there a way around that? using `defined` would really be a last resort measurement